flink-connector-aws/flink-connector-aws-kinesis-streams/src/main/java/org/apache/flink/connector/kinesis/source/config/KinesisSourceConfigOptions.java

```diff
@@ -19,12 +19,17 @@
 package org.apache.flink.connector.kinesis.source.config;
 
 import org.apache.flink.annotation.Experimental;
+import org.apache.flink.annotation.PublicEvolving;
 import org.apache.flink.configuration.ConfigOption;
 import org.apache.flink.configuration.ConfigOptions;
+import org.apache.flink.connector.aws.config.AWSConfigOptions;
+
+import java.time.Duration;
 
 /** Constants to be used with the KinesisStreamsSource. */
 @Experimental
-public class KinesisStreamsSourceConfigConstants {
+@PublicEvolving
```

Comment:
Hmmm, there's no point having both here - Only 1 is required. Because this connector is still `@Experimental` we can keep it that way 👀